Demo-Code.py

```diff
@@ -26,7 +41,15 @@ def get_random_pic(max_balls, min_diam, max_diam, bg_width, bg_height):
     ball_list.sort(key = lambda item: item[0] + item[1])
```

Comment:
Usually when using keyword arguments the equal sign is not spaced
```suggestion
    ball_list.sort(key=lambda item: item[0] + item[1])
```